addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -524,7 +524,7 @@ Note that transformation is only one part of the overall process how polled data
 Consult [Read steps](#read-steps) and [Write steps](#write-steps) for more details.
 Specifically, note that you might not need transformations at all in some uses cases.
 
-Please also note that you should install relevant transformations, as necessary. For example, `openhab-transformation-javascript` feature provides the javascript (`JS`) transformation.
+Please also note that you should install relevant transformations in openHAB as necessary. For example, `openhab-transformation-javascript` feature provides the javascript (`JS`) transformation.
```

Comment:
```suggestion
Please also note that you should install relevant transformations in openHAB as necessary.
For example, `openhab-transformation-javascript` feature provides the JavaScript (`JS`) transformation.
```

Reply:
@davidgraeff Please don't get me wrong. The main reason for this suggestion was the incorrect spelling of JavaScript. Next time I will point it out more clearly. :wink: